src/transformers/trainer.py

```diff
@@ -3423,28 +3421,25 @@ def _save_tpu(self, output_dir: Optional[str] = None):
                 self.accelerator.unwrap_model(model).save_pretrained(
                     output_dir,
                     is_main_process=self.args.should_save,
-                    state_dict=model.state_dict(),
+                    state_dict=xm._maybe_convert_to_cpu(model.state_dict()),
```

Comment:
I'm concerned by the fact we need to rely on a private `_maybe_convert_to_cpu` func here, what version of torch_xla was this introduced? Recently?

(We need to be careful with that, even if it's still broken for users not on that xla version)

Reply:
This func has always been present in torch_xla, and it should have been available as early as version 1.5.0.